IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl_demo/wolfssl_demo.c

```diff
@@ -294,78 +339,126 @@ static void Tls_client()
     /* attempt to connect TLS server */
 
     PeerAddr.sin_addr = FreeRTOS_inet_addr(TLSSERVER_IP);
-    PeerAddr.sin_port = FreeRTOS_htons(TLSSERVER_PORT);
+    PeerAddr.sin_port = FreeRTOS_htons(p->port);
 
     ret = FreeRTOS_connect(socket, &PeerAddr, sizeof(PeerAddr));
 
     if (ret != 0) {
-        printf("ERROR FreeRTOS_connect: %d\n",ret);
+        msg(pcName, p->id, "ERROR FreeRTOS_connect: %d\n",ret);
+        ret = -1;
+    }
+
+   #if defined(TLS_MULTITHREAD_TEST)
+    msg(pcName, p->id, " Ready to connect.\n");
+    xStatus = xSemaphoreTake(p->xBinarySemaphore, portMAX_DELAY);
+    if (xStatus != pdTRUE) {
+        msg(pcName, p->id, " Error : Failed to xSemaphoreTake\n");
+        goto out;
     }
+   #endif
 
     /* create WOLFSSL object */
     if (ret == 0) {
         ssl = wolfSSL_new(ctx);
         if (ssl == NULL) {
-            printf("ERROR wolfSSL_new: %d\n", wolfSSL_get_error(ssl, 0));
+            msg(pcName, p->id, "ERROR wolfSSL_new: %d\n", 
+                                        wolfSSL_get_error(ssl, 0));
             ret = -1;
         }
     }
+
     if (ret == 0) {
         #ifdef WOLFSSL_RENESAS_TSIP_TLS
-        tsip_set_callback_ctx(ssl, &userContext);
+         #if !defined(TLS_MULTITHREAD_TEST)
```

Comment:
Tab indents should be 4 spaces, not 1.  Please adjust indentation in this section of code.

Reply:
Thanks. Fixed